controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -429,6 +442,13 @@ func (r *StorageClassRequestReconciler) reconcileCephFilesystemSubVolumeGroup()
 		r.cephFilesystemSubVolumeGroup.Spec = rookCephv1.CephFilesystemSubVolumeGroupSpec{
 			FilesystemName: cephFilesystem.Name,
 		}
+
+		if r.StorageClassRequest.Annotations[controllers.StorageRequestForceDeleteAnnotation] == "true" {
+			if r.cephFilesystemSubVolumeGroup.Annotations == nil {
+				r.cephFilesystemSubVolumeGroup.Annotations = make(map[string]string)
+			}
+			r.cephFilesystemSubVolumeGroup.Annotations[RookCephResourceForceDeleteAnnotation] = "true"
+		}
```

Comment:
There is an add annotation util that does all of this. No need to write this logic yourself.